src/schemas/AlertCreateInput.yaml

```diff
@@ -0,0 +1,26 @@
+type: object
+required:
+  - alert
+properties:
+  alert:
+    allOf:
+      - $ref: './AlertBaseInput.yaml'
```

Comment:
If the goal is to extend the `AlertBaseInput` I think you have to allign the next object with the `- $ref:`